src/mmw/apps/bigcz/views.py

```diff
@@ -87,7 +87,44 @@ def _do_search(request):
         raise ParseError(ex.message)
 
 
+def _get_details(request):
+    params = request.query_params
+    catalog = params.get('catalog')
+
+    if not catalog:
```

Comment:
I wonder if there should be some sort of request serializer that would handle all the validation here and [in here](https://github.com/WikiWatershed/model-my-watershed/pull/2328/commits/3cbed5c4798a1ee2916b20e32e9585ee1e504a8f#diff-cb0b2d859416dc6a0e55365632b75e0aR14). Validation errors off a serializer [would also be a list](http://www.django-rest-framework.org/api-guide/serializers/#validation), instead of the way it's done here, returning only one missing field error at a time.

This isn't that necessary at the moment, but could be something we add when we want to expose the API.